app/controllers/api/databases.php

```diff
@@ -233,38 +238,39 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
     ->label('sdk.response.code', Response::STATUS_CODE_OK)
     ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
     ->label('sdk.response.model', Response::MODEL_DATABASE_LIST)
+    ->param('queries', [], new Databases(), 'Array of query strings generated using the Query class provided by the SDK. [Learn more about queries](https://appwrite.io/docs/databases#querying-documents). Maximum of ' . APP_LIMIT_ARRAY_PARAMS_SIZE . ' queries are allowed, each ' . APP_LIMIT_ARRAY_ELEMENT_SIZE . ' characters long. You may filter on the following attributes: ' . implode(', ', Databases::ALLOWED_ATTRIBUTES), true)
```

Comment:
What about a description like "Array of query strings used to manipulate the list of results. [Learn more about queries.]" and mention the SDK helpers on the learn more page?

I only say this because the new `permissions` params fit the same case but we don't mention the SDK helper there